routing/unified_edges.go

```diff
@@ -250,7 +250,7 @@ func (u *edgeUnifier) getEdgeLocal(netAmtReceived lnwire.MilliSatoshi,
 		amt := netAmtReceived + lnwire.MilliSatoshi(inboundFee)
 
 		// Check valid amount range for the channel.
-		if !edge.amtInRange(amt) {
+		if htlcBlob.IsNone() && !edge.amtInRange(amt) {
```

Comment:
won't this HTLC be rejected by our downstream peer?

Reply:
If the downstream node isn't a node with custom channel support, then yes. But we'll only send out such HTLCs to custom channels, so the peers should understand the difference in value between the actual on-chain amount and what's potentially being transported in the HTLC. Added a comment to make more clear though.